guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -14,18 +14,20 @@ include::snip_firewalld.adoc[]
 [options="nowrap"]
 ----
 # firewall-cmd \
---add-port="53/udp" --add-port="53/tcp" \
---add-port="67/udp" \
---add-port="69/udp" \
---add-port="80/tcp" --add-port="443/tcp" \
+--add-service=dns \
+--add-service=dhcp \
+--add-service=tftp \
+--add-service=http \
+--add-service=https \
 ifdef::katello,satellite,orcharhino[]
 --add-port="5647/tcp" \
---add-port="8000/tcp" --add-port="9090/tcp" \
+--add-port="8000/tcp" \
+--add-port="9090/tcp" \
 endif::[]
 ifndef::katello,satellite,orcharhino[]
---add-port="8443/tcp" \
+--add-service=foreman-proxy \
 endif::[]
---add-port="8140/tcp"
+--add-service=puppetmaster
```

Comment:
Puppet is off by default in katello/satellite so I don't think we should open the port by default. Rather in the enabling Puppet procedure.